### PR TITLE
Removing the survey banner

### PR DESCRIPTION
**Pull request by olaaustine**
Removing the survey banner

### CODE REVIEW

File: templates/header.html.ep [hunk 1]
```diff
@@ -1,4 +1,3 @@
-%= include 'survey_services'
```
Comment (dglemos) [inline templates/header.html.ep:1]: Shouldn't also remove the template?
`templates/survey_services.html.ep`

Comment (olaaustine) [reply templates/header.html.ep:1]: I wondered if we should renove it (because of history purposes), I can remove it if you think it is a good idea.

Comment (dglemos) [reply templates/header.html.ep:1]: It's ok to remove it.
We have the PR to keep track.

Comment (olaaustine) [reply templates/header.html.ep:1]: Updated. Thank you

**Review by dglemos — APPROVED** [review]

**Comment by dglemos** [comment]
Merged into release/112 and master

**Comment by olaaustine** [comment]
Updated to the live site

**Pull request merged**